__tests__/app.test.js

```diff
@@ -1,8 +1,47 @@
 const request = require('supertest');
 const db = require('../db/connection');
 const seed = require('../db/seeds/seed');
-const testTopicData = require('../db/data/test-data/topics');
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index');
 const { app } = require("../app");
 
 afterAll(() => db.end());
-beforeEach( () => seed(testTopicData));
\ No newline at end of file
+beforeEach( () => seed({topicData, userData, articleData, commentData}));

```

Comment:
you can clean this up slightly by making the required data a single variable of just `testData` without the object brackets, and then put it straight in the seed like this: `beforeEach( () => seed(testData) );`